src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.ListView.cs

```diff
@@ -98,15 +98,12 @@ private void StartSeeking(int? frame, bool fromMiddleClick = false)
 			TastudioPlayMode(); // suspend rec mode until seek ends, to allow mouse editing
 			MainForm.UnpauseEmulator();
 
-			if (!_seekBackgroundWorker.IsBusy && diff > TasView.VisibleRows)

```

Comment:
This line was used to disable the progressbar if we're seeking through some short distance, because that usually gets edited the most, and the progressbar becomes distractive. And if we're seeking to a frame that's within the view, there's no useful info in the progressbar. Can you bring this back?

Reply:
Added back the relevant part of the conditional.  With that, I can't get the progress bar to ever show up in trivial testing scenarios now, but maybe you have bigger movies that you can still verify it with.

Reply:
Did you enable auto-restore while testing? If not, you'll need to press mouse wheen after editing a cell to make it restore playback position.